src/client/.eslintrc.json

```diff
@@ -14,6 +21,12 @@
   "rules": {
     "import/prefer-default-export": 0,
     "no-console": 0,
-    "require-jsdoc" : 0
+    "require-jsdoc": 0,
+    "semi": [
+      2,
+      "never"
+    ],
+    "no-invalid-this": 0,
```

Comment:
Is this rule required in order to add support for plugin-proposal-class-properties? Let me know -- if not would prefer to leave out specific linting decisions and let users decide on their preferences.

Reply:
If we don't use this rule, we will get an annoying linting issue like this:

![Screenshot 2019-07-11 10:20:47](https://user-images.githubusercontent.com/4060207/61019876-eb1de300-a3c5-11e9-9995-33817bdec331.png)

Reply:
got it, looks good